e2e/create.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { test, expect, Page } from '@playwright/test';
+import { GuidedFormCreation, Create } from '../packages/design/src/FormManager/routes';
+import { BASE_URL } from './constants';
+import html from 'astro/dist/vite-plugin-html';
+
+const uuidPattern = '[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}';
+
+const createNewForm = async (page: Page) => {
+  await page.goto(`${BASE_URL}/${GuidedFormCreation.getUrl()}`);
+  await page.getByRole('button', { name: 'Create New' }).click();
+}
+
+test('Create form from scratch', async ({ page }) => {
+  const createPage = new RegExp(`${uuidPattern}/${Create.slug}$`, 'i');
```

Comment:
I don't think we'd actually want to add a `react-router-dom` dependency to this module, but it has a `generatePath` function that could be used in lieu of adding the slug - https://reactrouter.com/en/main/utils/generate-path

On 10x ASAP, we used https://www.npmjs.com/package/path-to-regexp as a lightweight option.

Just thinking out loud...